src/pages/en/core-concepts/astro-components.md

```diff
@@ -348,18 +348,30 @@ They can be used to style your components, and all style rules are automatically
 
 To send JavaScript to the browser without [using a framework component](/en/core-concepts/framework-components) (React, Svelte, Vue, Preact, SolidJS, AlpineJS, Lit) or an [Astro integration](https://astro.build/integrations/) (e.g. astro-XElement), you can use a `<script>` tag in your Astro component template and send JavaScript to the browser that executes in the global scope.
 
+By default, `<script>` tags are hoisted and `type="module"`. They will be placed in the `<head>` of the document and will be bundled with your imports. This is useful for sending client-side scripts to the browser that are not tied to a specific component.
+
+> ⚠️ TypeScript support is not currently available for client-side scripts. Please a look at the [RFC](https://github.com/withastro/rfcs/discussions/193) for more information on the status of this feature.
+
 ```astro
 <script>
   // Processed! Bundled! ESM imports work, even to npm packages.
 </script>
+```
+
+To avoid bundling the script, you can use the `is:inline` attribute.
 
+```astro
 <script is:inline>
   // Will be rendered into the HTML exactly as written!
   // ESM imports will not be resolved relative to the file.
 </script>
 ```
 
-📚 See our [directives reference](/en/reference/directives-reference#script--style-directives) page for more information about the directives available  on `<script>` tags.
+Multiple `<script>` tags can be used on the same Astro component with methods above.
+
+> **Note :** Adding `type="module"` or any other attribute to the `<script>` tag will prevent it from being bundled by Astro.
```

Comment:
```suggestion
> **Note :** Adding`type="module"` or any other attribute to  a `<script>` tag removes the default hoist behaviour and  prevents it from being bundled by Astro.
```
I'm not sure if this is the best way to say "removes the hoist", but I think it's worth saying that:
- it removes the hoise
-  it's now just like you've added is:inline
Can you think of the best way to get those words in there?  :)

Reply:
```suggestion
> **Note :** Adding `type="module"` or any other attribute to a `<script>` tag will treats it it's `is:inline` directive and remove it's default `hoist` behavior, disabling bundling for this specific tag.
```

More like this ?

Reply:
Yes, more like that.  :)  I'm going to ask @delucis to help with the exact wording, because my brain is a little mushy, but yes, I think that, and the rest of this, is exactly what we needed for the small scope of improving this page right now! 🥳

Reply:
```suggestion
> **Note:** Adding `type="module"` or any other attribute to a `<script>` tag will cause it to be treated as if it had an `is:inline` directive and remove its default behavior, disabling bundling for the tag.
```

Reply:
> **Note:** Adding `type="module"` or any other attribute to a `<script>` tag will **disable** Astro's default bundling behavior, treating it as if it had an `is:inline` directive.

Does this work?

**disable? override? revert? ignore?**

Reply:
💯 **disable** works perfectly \*chef’s kiss\*

Reply:
From the list, i prefer disable with the context given